src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -0,0 +1,514 @@
+> doi: [10.1177/0271678X20905433](https://doi.org/10.1177/0271678X20905433)
+
+The following example PET datasets have been formatted using this specification
+and can be used for practical guidance when curating a new dataset.
+
+-   One subject single dynamic scan (pet, mri, blood):
+	[`pet_example1`](https://www.dropbox.com/sh/dmjmaxfswatptpm/AACw31cTyChLA1kwM4Vhqueja?dl=0)
+-   One subject single dynamic scan (pet, mri):
+	[`pet_example2`](https://www.dropbox.com/sh/i95a71rpq2unks7/AACby5XcYQp9vcwgx3cNXBhza?dl=0)
+-   One subject single dynamic scan (pet, blood):
+	[`pet_example3`](https://www.dropbox.com/sh/i0c3iq50id4s0d3/AABSjaczHM1SQnMRvh3fc7fna?dl=0)
+
+Further PET datasets are available from [OpenNeuro](https://openneuro.org).
+
+## PET recording data
+
+As specified above, this extension is relevant for brain PET imaging
+and its associated data, such as blood data.
+In addition, the extension is in accordance with the guidelines for reporting
+and sharing brain PET data (Knudsen et al. 2020, [1]).
+
+*Shared MRI data along with PET:*
+To share structural magnetic resonance (MR) images with your PET data,
+please follow the original BIDS specification
+(https://bids-specification.readthedocs.io/en/stable).
+Please pay specific attention to the format the MR images are in,
+such as if they have been unwarped in order to correct for gradient non-linearities.
+There is a specific field in the MRI BIDS (https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/01-magnetic-resonance-imaging-data.html) called “NonlinearGradientCorrection”
+which indicates this.
+The reason for this is that the MRI needs to be corrected for nonlinear gradients
+in order to fit the accompanying PET scans for co-registration [1, 2].
+In general, SI units must be used (we refer to https://bids-specification.readthedocs.io/en/stable/99-appendices/05-units.html) and we recommend to use the CMIXF style formatting for SI units, for example, "kBq/mL"
+rather than "kilobecquerel per ml".
+An overview of a common PET experiment (with blood data) can be seen in Figure 1,
+defined on a single time scale relative to a predefined “time zero”
+(which should be defined either to be scan start or injection time;
+please note an exception to this definition is possible if a pharmacological
+within-scan challenge is performed).
+
+![Figure 1](images/PET_scan_overview.png "Overview of PET recording data")
+
+Figure 1: Overview of a common PET experiment, including blood measurements,
+and defined on a common time scale.
+Note, “time zero” can either be defined as time of injection or scan start,
+and all the PET and blood data should be decay-corrected to this time point.
+Furthermore, although in this example tracer injection coincides with scan start,
+this is not always the case and hence we allow for the flexibility of specifying
+either time of injection or scan start as “time zero”.
+
+Template:
```

Comment:
I would start the section with this.
This is what people expect to see when they reach the section. The other notes and comments could go either before (out of the section) or after. But the template is the most important and should be found at the beginning (This is also how it is in other specs)

Reply:
I moved this now.